pkgs/applications/graphics/rawtherapee/sigc++_fix.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/rtgui/adjuster.cc b/rtgui/adjuster.cc
```

Comment:
Is this your patch? If not, it is better to fetch patches from an external source rather than checking them into the repo. If it is your patch, consider contributing it upstream.